includes/admin/builder/class-evf-builder-fields.php

```diff
@@ -82,7 +82,8 @@ public function output_content() {
 		<div class="everest-forms-preview-wrap">
 			<div class="everest-forms-preview">
 				<div class="everest-forms-title-desc">
-					<h2 class="everest-forms-form-name"><?php echo isset( $this->form->post_title ) ? esc_html( $this->form->post_title ) : esc_html__( 'Form not found.', 'everest-forms' ); ?></h2>
+					<input type="text" disabled autocomplete="off" required id= "evf-edit-form-name" class="everest-forms-form-name everest-forms-name-input" value ="<?php echo isset( $this->form->post_title ) ? esc_html( $this->form->post_title ) : esc_html__( 'Form not found.', 'everest-forms' ); ?>">
```

Comment:
Make sure inconsistent blank space is not present for attributes.

Reply:
Noted, they're taken care of.